tests/jvm/src/test/scala/scala/meta/tests/prettyprinters/RegressionSyntaxSuite.scala

```diff
@@ -0,0 +1,25 @@
+package scala.meta.tests.prettyprinters
+
+import scala.meta.tests.parsers.ParseSuite
+
+import scala.meta._
+import munit.TestOptions
+
+class RegressionSyntaxSuite extends ParseSuite {
```

Comment:
do we implement my `forceUnderscore` suggestion, should we add a test parsing one dialect and printing another?